roxctl/central/crs/generate.go

```diff
@@ -79,11 +79,14 @@ func generateCRS(cliEnvironment environment.Environment, name string,
 
 	cliEnvironment.Logger().InfofLn("Successfully generated new CRS")
 	cliEnvironment.Logger().InfofLn("")
-	cliEnvironment.Logger().InfofLn("  Name:       %s", meta.GetName())
-	cliEnvironment.Logger().InfofLn("  Created at: %s", meta.GetCreatedAt().AsTime().Format(time.RFC3339))
-	cliEnvironment.Logger().InfofLn("  Expires at: %s", meta.GetExpiresAt().AsTime().Format(time.RFC3339))
-	cliEnvironment.Logger().InfofLn("  Created By: %s", getPrettyUser(meta.GetCreatedBy()))
-	cliEnvironment.Logger().InfofLn("  ID:         %s", meta.GetId())
+	cliEnvironment.Logger().InfofLn("  Name:               %s", meta.GetName())
+	cliEnvironment.Logger().InfofLn("  Created at:         %s", meta.GetCreatedAt().AsTime().Format(time.RFC3339))
+	cliEnvironment.Logger().InfofLn("  Expires at:         %s", meta.GetExpiresAt().AsTime().Format(time.RFC3339))
+	cliEnvironment.Logger().InfofLn("  Created By:         %s", getPrettyUser(meta.GetCreatedBy()))
+	if meta.GetMaxRegistrations() > 0 {
+		cliEnvironment.Logger().InfofLn("  Registration limit: %d clusters", meta.GetMaxRegistrations())
+	}
+	cliEnvironment.Logger().InfofLn("  ID:                 %s", meta.GetId())
```

Comment:
It would be great if you could add unit tests for this function
Here is an example how to mock grpc service https://github.com/stackrox/stackrox/blob/46c3f6af956331854a86a0c235c3017c3f0b39f1/roxctl/central/whoami/whoami_test.go#L53